lib/DWARFImporter/DWARFImporter.cpp

```diff
@@ -13,33 +13,37 @@
 #include "swift/DWARFImporter/DWARFImporter.h"
 #include "swift/AST/ASTContext.h"
 #include "swift/AST/Module.h"
+#include "clang/CodeGen/ObjectFilePCHContainerOperations.h"
+#include "clang/Frontend/FrontendActions.h"
+#include "../ClangImporter/ImporterImpl.h"
```

Comment:
This is very questionable. ClangImporter::Implementation isn't designed for reuse; the distribution of information between ClangImporter and ClangImporter::Implementation is basically "whatever's most convenient". If we need a common base class, let's factor that out and put it in the include/ folder.